test/Microsoft.TestPlatform.Utilities.UnitTests/CodeCoverageRunSettingsProcessorTests.cs

```diff
@@ -0,0 +1,308 @@
+                @"        <Function>^Microsoft::VisualStudio::CppCodeCoverageFramework::.*</Function>",
+                @"        <Function>^Microsoft::VisualStudio::CppUnitTestFramework::.*</Function>",
+                @"        <Function>.*::YOU_CAN_ONLY_DESIGNATE_ONE_.*</Function>",
+                @"        <Function>^__.*</Function>",
+                @"        <Function>.*::__.*</Function>",
+                @"      </Exclude>",
+                @"    </Functions>",
+                @"    <Attributes>",
+                @"      <Exclude>",
+                @"        <Attribute>^System.Diagnostics.DebuggerHiddenAttribute$</Attribute>",
+                @"        <Attribute>^System.Diagnostics.DebuggerNonUserCodeAttribute$</Attribute>",
+                @"        <Attribute>System.Runtime.CompilerServices.CompilerGeneratedAttribute$</Attribute>",
+                @"        <Attribute>^System.CodeDom.Compiler.GeneratedCodeAttribute$</Attribute>",
+                @"        <Attribute>^System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute$</Attribute>",
+                @"        <Attribute>^Microsoft.VisualStudio.TestPlatform.TestSDKAutoGeneratedCode.*</Attribute>",
+                @"      </Exclude>",
+                @"    </Attributes>",
+                @"    <Sources>",
+                @"      <Exclude>",
+                @"          <Source>.*\\atlmfc\\.*</Source>",
+                @"          <Source>.*\\atlmbfc\\.*</Source>",
+                @"          <Source>.*\\vctools\\.*</Source>",
+                @"          <Source>.*\\public\\sdk2\\.*</Source>",
+                @"          <Source>.*\\externalapis\\.*</Source>",
+                @"          <Source>.*\\microsoft sdks\\.*</Source>",
+                @"          <Source>.*\\vc\\include\\.*</Source>",
+                @"          <Source>.*\\msclr\\.*</Source>",
+                @"          <Source>.*\\ucrt\\.*</Source>",
+                @"          <Source>.*\\public\\sdk\\.*</Source>",
+                @"      </Exclude>",
+                @"    </Sources>",
+                @"  </CodeCoverage>",
+                @"</Configuration>");
+
+            var expectedResultDocument = new XmlDocument();
+            expectedResultDocument.LoadXml(expectedResult);
+
+            var processedNode = processor.Process(settings);
+            Assert.IsNotNull(processedNode);
+
+            var codeCoverageNodes = this.ExtractNodes(processedNode, expectedResultDocument.DocumentElement, "./CodeCoverage");
+
+            this.CompareResults(codeCoverageNodes.Item1, codeCoverageNodes.Item2, "./ModulePaths/Exclude");
+            this.CompareResults(codeCoverageNodes.Item1, codeCoverageNodes.Item2, "./Functions/Exclude");
+            this.CompareResults(codeCoverageNodes.Item1, codeCoverageNodes.Item2, "./Attributes/Exclude");
+            this.CompareResults(codeCoverageNodes.Item1, codeCoverageNodes.Item2, "./Sources/Exclude");
+        }
```

Comment:
please add tests for mergeDefaults = false attribute

Reply:
Done.